execution-engine/src/fs.rs

```diff
@@ -1951,12 +1971,9 @@ impl FileSystem {
     }
 
     /// Check if a `file_name` exists.
-    /// Note: this function *has* side effect ! 
+    /// Note: this function *has* side effect !
```

Comment:
(Space before exclamation mark is very French.)

Reply:
It is. @ShaleXIONG are you a French spy? Speculatively marking this as resolved but waiting for a background check on @ShaleXIONG